Detectors/TPC/reconstruction/src/TPCCATracking.cxx

```diff
@@ -244,9 +244,17 @@ int TPCCATracking::runTracking(const ClusterNativeAccessFullTPC& clusters, std::
           
       if (!mTrackingCAO2Interface->GetParamContinuous())
         oTrack.setTime0(0);
-      else
-        oTrack.setTime0(sContinuousTFReferenceLength - (tracks[i].Side() ? 1.f : -1.f) * tracks[i].GetParam().GetZOffset() / (elParam.getZBinWidth() * gasParam.getVdrift()));
+      else {
+        float zoffset = tracks[i].CSide() ?  -tracks[i].GetParam().GetZOffset() : tracks[i].GetParam().GetZOffset();
```

Comment:
can this be a potential problem since `Side::UNDEFINED` and `Side::C` are treated the same in this condition?